data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -70,14 +71,21 @@ is_api_available() {
 	echo "$output"
 	return 1
     fi
+
+    threshold=2
+    if [ "$BOOTSTRAP_INPLACE" = true ]
+    then
+	threshold=1
+    fi
+    echo "Need at least $threshold available addresses for the default/kubernetes service"
     
     echo "Got the following addresses for the default/kubernetes endpoint object: $output"
     count=$(echo "$output" | wc -w)
-    if [[ ! $count -gt 1 ]]
+    if [[ ! $count -gt $threshold ]]
     then
 	return 1
     fi
     
-    echo "Got at least 2 available addresses for the default/kubernetes service"
+    echo "Got at least $threshold available addresses for the default/kubernetes service"
```

Comment:
```suggestion
    echo "Got at least $(($threshold+1)) available addresses for the default/kubernetes service"
```
Or maybe make this a variable